collector/slave_hosts.go

```diff
@@ -31,12 +31,12 @@ const (
 	// timestamps. %s will be replaced by the database and table name.
 	// The second column allows gets the server timestamp at the exact same
 	// time the query is run.
-	slaveHostsQuery = "SHOW SLAVE HOSTS"
 )
 
 // Metric descriptors.
 var (
-	SlaveHostsInfo = prometheus.NewDesc(
+	slaveHostsQueries = [2]string{"SHOW SLAVE HOSTS", "SHOW REPLICAS"}
```

Comment:
I would rather keep the const and add a new `showReplicasQuery` const for the new version.